# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-gcc15-gnu17.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./Makefile.vanilla	2025-04-27 20:33:18.044238444 +0200
+++ ./Makefile	2025-04-27 20:34:20.824704820 +0200
@@ -592,10 +592,10 @@
 		    -Wclippy::float_arithmetic
 
 KBUILD_AFLAGS_KERNEL :=
-KBUILD_CFLAGS_KERNEL :=
+KBUILD_CFLAGS_KERNEL := -std=gnu17
 KBUILD_RUSTFLAGS_KERNEL :=
 KBUILD_AFLAGS_MODULE  := -DMODULE
-KBUILD_CFLAGS_MODULE  := -DMODULE
+KBUILD_CFLAGS_MODULE  := -std=gnu17 -DMODULE
 KBUILD_RUSTFLAGS_MODULE := --cfg MODULE
 KBUILD_LDFLAGS_MODULE :=
 KBUILD_LDFLAGS :=
